README.md

```diff
@@ -39,6 +39,7 @@ Creates new BrowserStack workers and returns an array of workers ids.
 		* `browser`: The browser name
 		* `browser_version`: The browser version
 		* `device`: The device name
+		* `worker_name`: The worker name
```

Comment:
I suppose we should check this description with a tech writer (maybe we should write a little more here or it's ok)